Corrections/hemispherize.cc

```diff
@@ -2,14 +2,19 @@
 
 #include <cmath>
 using namespace ROOT::VecOps;
-using rvec_f = const RVec<float> &;
-using rvec_i = const RVec<int> &;
+using rvec_f = const RVec<float>;
+using rvec_i = const RVec<int>;
+
 //return two ak4 cnadidates that are properly selected by hemispherize funtion for 2+1
 //Compares ak4 jets against leading ak8 and looks for any in opposite hemisphere
 //First find the highest pt ak8 jet with mass > 40 geV
 namespace analyzer {
-     RVec<int> Hemispherize(rvec_f FJpt, rvec_f FJeta, rvec_f FJphi, rvec_f FJmass, rvec_i FJnjets, rvec_f Jpt, rvec_f Jeta, rvec_f Jphi, rvec_f Jmass, rvec_i Jnjets, rvec_f btagDeepB){
+     RVec<int> Hemispherize(rvec_f FJpt, rvec_f FJeta, rvec_f FJphi, rvec_f FJmass, int FJnjets, rvec_f Jpt, rvec_f Jeta, rvec_f Jphi, rvec_f Jmass, int Jnjets, rvec_f btagDeepB){
```

Comment:
It sucks we have to declare so many arguments like this. We should think about each JHUanalyzer-provided c++ module having a python wrapper that the user can easily call and give just an argument or two and then that wrapper handles all of the input to this messy c++ function.